project.yml

```diff
@@ -50,7 +50,7 @@ packages:
     maxVersion: 2.0.0
   SwiftOGG:
     url: https://github.com/vector-im/swift-ogg
-    branch: main
+    branch: 0.0.1
```

Comment:
Shouldn't this also have updated `Package.resolved`? 🤔

Reply:
Ah I missed it, thanks!